--- HDF4.2r1/config/linux-gnu.orig	2004-12-08 00:04:44.000000000 +0100
+++ HDF4.2r1/config/linux-gnu	2005-02-27 21:30:49.561220144 +0100
@@ -120,7 +120,7 @@
     CFLAGS="$CFLAGS"
     DEBUG_CFLAGS="-g -fverbose-asm"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O3 -fomit-frame-pointer"
+    PROD_CFLAGS="-fomit-frame-pointer"
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
     PROFILE_CPPFLAGS=
@@ -130,7 +130,7 @@
     CFLAGS="$CFLAGS"
     DEBUG_CFLAGS="-g"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O"
+    PROD_CFLAGS=""
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
     PROFILE_CPPFLAGS=
@@ -141,7 +141,7 @@
   g77)
     FFLAGS="$FFLAGS -Wsign-compare"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O3 -fomit-frame-pointer"
+    PROD_FFLAGS="-fomit-frame-pointer"
     PROFILE_FFLAGS="-pg"
     ;;
 
@@ -149,7 +149,7 @@
     CFLAGS="$CFLAGS -DH4_ABSOFT"
     FFLAGS="$FFLAGS -YEXT_NAMES=UCS"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
+    PROD_FFLAGS=""
     PROFILE_FFLAGS="-pg"
     ;;
 esac
