--- hdf-4.2.15/configure.ac.orig	2020-11-12 21:00:56.401745032 +0100
+++ hdf-4.2.15/configure.ac	2020-11-12 21:04:32.302866945 +0100
@@ -533,7 +533,7 @@
   X-yes)
     HAVE_SZIP="yes"
     AC_CHECK_HEADERS([szlib.h], [HAVE_SZLIB_H="yes"], [unset HAVE_SZIP])
-    if test "x$HAVE_SZLIB" = "xyes" -a "x$HAVE_SZLIB_H" = "xyes"; then
+    if test "x$HAVE_SZIP" = "xyes" -a "x$HAVE_SZLIB_H" = "xyes"; then
       AC_CHECK_LIB([sz], [SZ_BufftoBuffCompress],, [unset HAVE_SZIP])
     fi
 
