--- hdf-4.2.16-2/config/gnu-flags.orig	2024-12-18 19:47:50.539445329 +0100
+++ hdf-4.2.16-2/config/gnu-flags	2024-12-18 19:47:53.502762608 +0100
@@ -120,7 +120,7 @@ if test "X-gcc" = "X-$cc_vendor"; then
     fi
     DEBUG_CFLAGS="-g -fverbose-asm"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O3 -fomit-frame-pointer"
+    PROD_CFLAGS="-fomit-frame-pointer"
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
     PROFILE_CPPFLAGS=
--- hdf-4.2.16-2/config/gnu-fflags.orig	2023-06-27 17:28:51.000000000 +0200
+++ hdf-4.2.16-2/config/gnu-fflags	2024-12-18 19:48:11.835996622 +0100
@@ -62,14 +62,14 @@
 # value of CC may have changed above).
 
 case $F77_BASENAME in
-  gfortran)
+  *gfortran*)
     if test $cc_vers_major -ge 10; then
         FFLAGS="$FFLAGS -fallow-argument-mismatch"
     else
         FFLAGS="$FFLAGS"
     fi
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
+    PROD_FFLAGS=""
     PROFILE_FFLAGS="-pg"
     ;;
 
   g77)
     FFLAGS="$FFLAGS "
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O3 -fomit-frame-pointer"
+    PROD_FFLAGS="-fomit-frame-pointer"
     PROFILE_FFLAGS="-pg"
     ;;
 
@@ -84,7 +84,7 @@ case $F77_BASENAME in
     CFLAGS="$CFLAGS"
     FFLAGS="$FFLAGS"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
+    PROD_FFLAGS=""
     PROFILE_FFLAGS="-pg"
     ;;
 
@@ -92,7 +92,7 @@ case $F77_BASENAME in
     CFLAGS="$CFLAGS"
     FFLAGS="$FFLAGS -i4"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
+    PROD_FFLAGS=""
     PROFILE_FFLAGS="-pg"
     ;;
 esac
--- ../BUILD/hdf-4.2.16-2/config/gnu-fflags.orig	2024-12-18 20:41:47.971906652 +0100
+++ ../BUILD/hdf-4.2.16-2/config/gnu-fflags	2024-12-18 20:54:55.677639283 +0100
