--- HDF4.2r3/mfhdf/hdiff/Makefile.am.orig	2008-06-04 21:11:34.882851763 +0200
+++ HDF4.2r3/mfhdf/hdiff/Makefile.am	2008-06-04 21:27:20.684746589 +0200
@@ -20,7 +20,7 @@
 hdiff_SOURCES = hdiff.c hdiff_array.c hdiff_gr.c hdiff_list.c hdiff_main.c  \
                 hdiff_mattbl.c hdiff_gattr.c hdiff_misc.c hdiff_sds.c       \
                 hdiff_table.c hdiff_vs.c hdiff_dim.c
-hdiff_LDADD = ../libsrc/libmfhdf.la $(top_builddir)/hdf/src/libdf.la @LIBS@
+hdiff_LDADD = ../libsrc/libmfhdf.la $(top_builddir)/hdf/src/libdf.la @LIBS@ -lm
 hdiff_DEPENDENCIES = $(top_builddir)/hdf/src/libdf.la ../libsrc/libmfhdf.la
 
 #############################################################################
