--- HDF4.2r4/configure.ac.orig	2009-01-13 22:13:49.000000000 +0100
+++ HDF4.2r4/configure.ac	2009-04-23 23:36:13.513713807 +0200
@@ -190,12 +190,6 @@
     enable_shared="no"
 fi
 
-if test "X${enable_shared}" = "Xyes"; then
-    if test "X${BUILD_FORTRAN}" = "Xyes"; then
-        AC_MSG_ERROR([Cannot build shared fortran libraries. Please configure with --disable-fortran flag.])
-    fi
-fi
-
 AC_PROG_INSTALL
 AM_PROG_LEX
 AC_PROG_INSTALL
