--- HDF4.2r1/config/linux-gnu.orig	2004-12-08 00:04:44.000000000 +0100
+++ HDF4.2r1/config/linux-gnu	2005-02-27 21:30:49.561220144 +0100
@@ -120,7 +120,7 @@
     CFLAGS="$CFLAGS"
     DEBUG_CFLAGS="-g -fverbose-asm"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O3 -fomit-frame-pointer"
+    PROD_CFLAGS="-fomit-frame-pointer"
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
     PROFILE_CPPFLAGS=
@@ -130,7 +130,7 @@
     CFLAGS="$CFLAGS"
     DEBUG_CFLAGS="-g"
     DEBUG_CPPFLAGS=
-    PROD_CFLAGS="-O"
+    PROD_CFLAGS=""
     PROD_CPPFLAGS=
     PROFILE_CFLAGS="-pg"
     PROFILE_CPPFLAGS=
@@ -141,7 +141,7 @@
   g77)
     FFLAGS="$FFLAGS -Wsign-compare"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O3 -fomit-frame-pointer"
+    PROD_FFLAGS="-fomit-frame-pointer"
     PROFILE_FFLAGS="-pg"
     ;;
 
@@ -149,7 +149,7 @@
     CFLAGS="$CFLAGS -DH4_ABSOFT"
     FFLAGS="$FFLAGS -YEXT_NAMES=UCS"
     DEBUG_FFLAGS="-g"
-    PROD_FFLAGS="-O"
+    PROD_FFLAGS=""
     PROFILE_FFLAGS="-pg"
     ;;
 esac
--- hdf-4.2.6/configure.ac.orig	2011-06-26 10:00:09.921294977 +0200
+++ hdf-4.2.6/configure.ac	2011-06-26 10:25:04.111345008 +0200
@@ -808,7 +808,7 @@
 dnl Report anything stripped as a flag in CFLAGS and 
 dnl only the compiler in CC_VERSION.
 CC_NOFLAGS=`echo $CC | sed 's/ -.*//'`
-CFLAGS_TO_ADD=`echo $CC | grep - | sed 's/.* -/-/'`
+CFLAGS_TO_ADD=`echo $CC | grep ' -' | sed 's/.* -/-/'`
 if test -n $CFLAGS_TO_ADD; then
   CFLAGS="$CFLAGS_TO_ADD$CFLAGS"
 fi
@@ -836,7 +836,7 @@
 dnl Report anything stripped as a flag in RFLAGS and 
 dnl only the compiler in F77_VERSION.
 F77_NOFLAGS=`echo $F77 | sed 's/ -.*//'`
-FFLAGS_TO_ADD=`echo $F77 | grep - | sed 's/.* -/-/'`
+FFLAGS_TO_ADD=`echo $F77 | grep ' -' | sed 's/.* -/-/'`
 if test -n $FFLAGS_TO_ADD; then
  FFLAGS="$FFLAGS_TO_ADD$FFLAGS"
 fi
