--- hdf4-hdf4.3.0/configure.ac.orig	2024-12-19 21:02:23.923808617 +0100
+++ hdf4-hdf4.3.0/configure.ac	2024-12-19 21:08:01.925310839 +0100
@@ -336,12 +336,6 @@ if test "X${enable_shared}" != "Xyes"; t
     enable_shared="no"
 fi
 
-if test "X${enable_shared}" = "Xyes"; then
-    if test "X${BUILD_FORTRAN}" = "Xyes"; then
-        AC_MSG_ERROR([Cannot build shared fortran libraries. Please configure with --disable-fortran flag.])
-    fi
-fi
-
 ## ----------------------------------------------------------------------
 ## Check if they would like the Java native interface (JNI) compiled
 ##
