--- hdf-4.2.5/hdf/src/hdfi.h.orig	2010-01-29 05:57:57.000000000 +0100
+++ hdf-4.2.5/hdf/src/hdfi.h	2010-10-09 08:33:07.215741882 +0200
@@ -155,7 +155,7 @@
  *      be renamed as LONGNEINT32.
  */
 
-#if (defined(SUN) || defined(sun) || defined(__sun__) || defined(__SUNPRO_C)) & !defined(__i386)
+#if (defined(SUN) || defined(sun) || defined(__sun__) || defined(__sparc__) || defined(__SUNPRO_C)) & !defined(__i386)
 #ifdef __STDC__
 #define ANSISUN
 #else /* __STDC__ */
@@ -282,7 +282,7 @@
 #endif /* SUN */
 
 
-#if defined(IBM6000) || defined(_AIX)
+#if defined(IBM6000) || defined(_AIX) || defined(__powerpc__)
 
 #ifndef IBM6000
 #define IBM6000
@@ -326,6 +326,9 @@
 typedef int               hdf_pint_t;   /* an integer the same size as a pointer */
 #endif /*AIX5L64 */
 #define _fcdtocp(desc) (desc)
+#ifdef __GNUC__
+#define FNAME_POST_UNDERSCORE
+#endif
 #define FILELIB UNIXBUFIO
 
 /* JPEG #define's - Look in the JPEG docs before changing - (Q) */
