--- hdf-4.2.16-2/mfhdf/libsrc/putget.c.orig	2023-02-06 15:13:34.000000000 +0100
+++ hdf-4.2.16-2/mfhdf/libsrc/putget.c	2024-12-19 18:01:49.998654094 +0100
@@ -593,7 +593,7 @@ xdr_NCv1data(XDR *xdrs, u_long where, nc
 #ifdef H4_HAVE_LP64
             return (xdr_int(xdrs, (nclong *)values));
 #else
-            return (xdr_long(xdrs, (nclong *)values));
+            return (xdr_long(xdrs, (long *)values));
 #endif
         case NC_FLOAT:
             return (xdr_float(xdrs, (float *)values));
