--- HDF4.2r1/hdf/src/cszip.c.orig	2005-01-08 21:12:34.000000000 +0100
+++ HDF4.2r1/hdf/src/cszip.c	2005-02-27 21:42:46.316256816 +0100
@@ -865,7 +865,9 @@
     compinfo_t *info;           /* special element information */
     comp_coder_szip_info_t *szip_info;    /* ptr to SZIP info */
 
+#ifdef H4_HAVE_LIBSZ
     if (SZ_encoder_enabled() == 0) 
+#endif
 	HRETURN_ERROR(DFE_NOENCODER, FAIL);
     info = (compinfo_t *) access_rec->special_info;
     szip_info = &(info->cinfo.coder_info.szip_info);
