--- hdf-4.2.15/configure.ac.orig	2020-03-03 18:40:50.000000000 +0100
+++ hdf-4.2.15/configure.ac	2020-11-12 21:00:42.794377396 +0100
@@ -294,12 +294,6 @@
     enable_shared="no"
 fi
 
-if test "X${enable_shared}" = "Xyes"; then
-    if test "X${BUILD_FORTRAN}" = "Xyes"; then
-        AC_MSG_ERROR([Cannot build shared fortran libraries. Please configure with --disable-fortran flag.])
-    fi
-fi
-
 AC_PROG_INSTALL
 AC_PROG_LN_S
 
